README.md

```diff
@@ -0,0 +1,273 @@
+```
+
+The `block_types` index was added in plone.volto 4.1.0.
+By default it is only added for new Plone sites.
+To add it to an existing site, run `plone.volto.upgrades.add_block_types_index` manually.
+
+
+### Multilingual support
+
+plone.volto supports multilingual websites.
+Install PAM before installing this package and demo homepages will be created in each enabled language.
+Currently only support for EN/DE.
+
+### Document content type
+
+plone.volto disables the `Richtext` and `Table of Contents` behaviors for the `Document` content type.
+Rich Text functionality is provided by the new Volto Blocks editor.
+The `Table of Contents` functionality is provided by the `Table of Contents Block` in Volto.
+
+
+### CORS profile
+
+A quick helper for enable CORS for development config is also provided in the
+`plone.volto` module. So you can call:
+
+```xml
+<include package="plone.volto.cors" />
+```
+
+from your ZCML while developing.
+
+Enable it on demand, since it's considered a security issue if you enable CORS in your
+productions sites.
+
+It's planned that Volto will feature a development pass-through proxy to the backend in
+the future. It will be addressed in next sprints.
+
+### ZLog patch
+
+p.restapi low level errors are routed through the ancient ZLog and are `plone_error`
+enabled, making it difficult to follow since all are marked with a UUID. Specially if
+using helpers like Sentry. This patch removes the UUID so the same error is categorized
+all together. This is planned to be addressed in next sprints.
+
+
+### Patch for `subject` field
+
+There are some problems of serialization on special characters derivated from how the
+current shape of the Plone's default Dexterity `subjects` field that has to be
+addressed in order to make it work properly with Volto (and other systems that are not
+Plone). This will be fixed in core in upcoming sprints.
+
+
+### Preview image behavior
+
+The preview image behavior makes content types provide a `preview_image` field that can store a preview image that Volto views can pick up.
+This is especially useful for listings (e.g. listing block customizations) and teaser elements (e.g. teaser blocks).
+
+The `volto.preview_image` behavior can be enabled in the generic setup XML definition of a content type (e.g. `/profiles/default/types/MyContentType.xml`):
+
+```xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<object i18n:domain="my.project" meta_type="Dexterity FTI" name="MyContentType"
+  xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+
+  ...
+
+  <!-- Enabled behaviors -->
+  <property name="behaviors" purge="false">
+    ...
+    <element value="volto.preview_image" />
+  </property>
+  ...
+</object>
+```
+
+There is also another variation of the preview image behavior called `volto.preview_image_link`.
+This one stores preview images using a relation to an Image content type, rather than in an image field. This might be preferable if many content items use the same preview image.
```

Comment:
```suggestion
This one stores preview images using a relation to an image content type, rather than in an image field. This might be preferable if many content items use the same preview image.
```